docs/README.md

```diff
@@ -26,92 +26,183 @@
-      - [Stateful Models](architecture.md#stateful-models)
-      - [Control Inputs](architecture.md#control-inputs)
-      - [Implicit State - Stateful Inference Using a Stateless Model](architecture.md#implicit-state-management)
-      - [Sequence Scheduling Strategies](architecture.md#scheduling-strategies)
-        - [Direct](architecture.md#direct)
-        - [Oldest](architecture.md#oldest)
-    - [Rate Limiter](rate_limiter.md)
-  - [Model Warmup](model_configuration.md#model-warmup)
-  - [Inference Request/Response Cache](model_configuration.md#response-cache)
-- Model Pipeline
-  - [Model Ensemble](architecture.md#ensemble-models)
-  - [Business Logic Scripting (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting)
-- [Model Management](model_management.md)
-  - [Explicit Model Loading and Unloading](model_management.md#model-control-mode-explicit)
-  - [Modifying the Model Repository](model_management.md#modifying-the-model-repository)
-- [Metrics](metrics.md)
-- [Framework Custom Operations](custom_operations.md)
-  - [TensorRT](custom_operations.md#tensorrt)
-  - [TensorFlow](custom_operations.md#tensorflow)
-  - [PyTorch](custom_operations.md#pytorch)
-  - [ONNX](custom_operations.md#onnx)
-- [Client Libraries and Examples](https://github.com/triton-inference-server/client)
-  - [C++ HTTP/GRPC Libraries](https://github.com/triton-inference-server/client#client-library-apis)
-  - [Python HTTP/GRPC Libraries](https://github.com/triton-inference-server/client#client-library-apis)
-  - [Java HTTP Library](https://github.com/triton-inference-server/client/tree/main/src/java)
-  - GRPC Generated Libraries
-    - [go](https://github.com/triton-inference-server/client/tree/main/src/grpc_generated/go)
-    - [Java/Scala](https://github.com/triton-inference-server/client/tree/main/src/grpc_generated/java)
-    - [Javascript](https://github.com/triton-inference-server/client/tree/main/src/grpc_generated/javascript)
-- Performance Analysis
-  - [Performance Tuning Guide](performance_tuning.md)
-  - [Optimization](optimization.md)
-  - [Model Analyzer](model_analyzer.md)
-  - [Performance Analyzer](perf_analyzer.md)
-  - [Inference Request Tracing](trace.md)
-- [Jetson and JetPack](jetson.md)
-
-## Developer Guide
-The Developer Guide describes how to build and test Triton and also how Triton can be extended with new functionality.
-
-- [Build](build.md)
-- [Protocols and APIs](inference_protocols.md).
+# **Triton Inference Server Documentation**
+
+| [Installation](README.md#installation) | [Getting Started](README.md#getting-started) | [User Guide](README.md#user-guide) | [API Guide](protocol/README.md) | [Additional Resources](README.md#resources) | [Customization Guide](README.md#customization-guide) |
+| ------------ | --------------- | --------------- | ------------ | --------------- | --------------- | 
+
+## **Installation**
+Before you can use the Triton Docker image you must install
```

Comment:
Nit for readability:
"Before you can use the Triton Docker image, you must..."
"If you plan on using a GPU for inference, you must also..."

Feel free to ignore if you think it reads better the current way.

Reply:
Docker can be use for CPU as well, so the initial comment is correct.